src/types/models/ObjectModel.jl

```diff
@@ -35,18 +32,12 @@ keys(model.reactions)
 # Fields
 $(TYPEDFIELDS)
 """
-mutable struct ObjectModel <: AbstractMetabolicModel
+Bas.@kwdef mutable struct ObjectModel <: AbstractMetabolicModel
     id::String
-    reactions::OrderedDict{String,Reaction}
-    metabolites::OrderedDict{String,Metabolite}
-    genes::OrderedDict{String,Gene}
-
-    ObjectModel(
-        id = "";
-        reactions = OrderedDict{String,Reaction}(),
-        metabolites = OrderedDict{String,Metabolite}(),
-        genes = OrderedDict{String,Gene}(),
-    ) = new(id, reactions, metabolites, genes)
+    reactions::OrderedDict{String,Reaction} = OrderedDict{String,Reaction}()
+    metabolites::OrderedDict{String,Metabolite} = OrderedDict{String,Metabolite}()
+    genes::OrderedDict{String,Gene} = OrderedDict{String,Gene}()
+    objective::Maybe{Tuple{String, Float64}} = nothing
```

Comment:
what about multi-reaction objectives?

Reply:
changed to `Maybe{Vector{Tuple{String, Float64}}}`

Reply:
So why not `Dict{String,Float64}` ?

Reply:
yeah, that is better